stock_location_tank/models/stock_picking.py

```diff
@@ -0,0 +1,182 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    production_ids = fields.One2many(
+        comodel_name="mrp.production",
+        inverse_name="picking_id",
+        string="Manufacturing Orders",
```

Comment:
If this is only for a liquid production it should be named `liquid_mixing_production_id`, `production_ids` is too common